MediaContentSynchronizationCms/etc/di.xml

```diff
@@ -14,6 +14,22 @@
             </argument>
         </arguments>
     </type>
+    <type name="Magento\MediaContentSynchronization\Model\RemoveObsoleteContentAsset">
+        <arguments>
+            <argument name="entities" xsi:type="array">
+                <item name="cms_block" xsi:type="string">cms_block</item>
+                <item name="cms_page" xsi:type="string">cms_page</item>
+            </argument>
+             <argument name="entityTableNames" xsi:type="array">
```

Comment:
I think it's better to use the `Magento\Framework\EntityManager\MetadataPool` to retrieve the table and identifier.
Yes, the `EntityManager` is not the first priority to use, but in this case, I'd say, it's just essential

Reply:
:+1:  ok